Authentication/AuthenticationBootstrapper.swift

```diff
@@ -75,49 +76,87 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
             _window.rootViewController = UINavigationController(rootViewController: createLoginController())
         }
     }
+}
 
-    /**
+// MARK: - Login Functions
+public extension AuthenticationBootstrapper {
+    
+    /*
+         Creates the login controller to use for starting
+         the authentication process.
+         
+         - Returns: A valid login controller to use.
+     */
+    internal func createLoginController() -> LoginController {
+        let configuration = createLoginControllerConfiguration()
+        return LoginController(configuration: configuration)
+    }
+    
+    /*
         Creates the log in credential validator that embodies what must be met
         so as to enable the log in for the user.
 
         - Returns: A log in credentials validator to use for creating the LogInViewModel.
 
         - Attention: Override this method for customizing the criteria of email and password validity.
+        You can use email as username if you use the correct validator for it.
     */
     public func createLogInCredentialsValidator() -> LoginCredentialsValidator {
         return LoginCredentialsValidator()
     }
 
-    /**
-         Creates the LogInViewModel to use in the authentication process logic,
+    /*
```

Comment:
Why did you change this? If you remove the second *, the user of this class/function will not be able to see the description.